packages/sdk/src/modules/Flywheel.ts

```diff
@@ -121,7 +121,8 @@ export function withFlywheel<TBase extends CreateContractsModule = CreateContrac
       for (const flywheel of flywheels) {
```

Comment:
you don't need to have 2 nested for loops, just call `getClaimRewardsForMarkets` with all `marketsOfPool` and all `flywheels` as input params - the result is return for each reward token of the flywheel in the input